actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -52,7 +52,7 @@ def self.define_element(name, code_generator:, method_name: name.to_s.underscore
           code_generator.define_cached_method(method_name, namespace: :tag_builder) do |batch|
             batch.push(<<~RUBY) unless instance_methods.include?(method_name.to_sym)
               def #{method_name}(content = nil, escape: true, **options, &block)
-                tag_string(#{name.inspect}, content, escape: escape, **options, &block)
+                tag_string("#{name.name}", content, options, escape: escape, &block)
```

Comment:
```suggestion
                tag_string("#{name}", content, options, escape: escape, &block)
```

Ruby interpolation is smart enough not to allocate when concatenating a symbol.

Reply:
TIL, that's much nicer